pkg/util/resource.go

```diff
@@ -77,27 +77,31 @@ func MinResourceList(a corev1.ResourceList, b corev1.ResourceList) corev1.Resour
 	return result
 }
 
-// IsResourceListEqualValue checks if the two resource lists are numerically equivalent.
-// NOTE: Resource name with a zero value will be ignored in comparison.
-// e.g. a = {"cpu": "10", "memory": "0"}, b = {"cpu": "10"} => true
-func IsResourceListEqualValue(a, b corev1.ResourceList) bool {
-	a = quotav1.RemoveZeros(a)
-	b = quotav1.RemoveZeros(b)
-	if len(a) != len(b) { // different number of no-zero resources
+// IsResourceListEqual checks if the two resource lists are numerically equivalent.
+func IsResourceListEqual(a corev1.ResourceList, b corev1.ResourceList) bool {
```

Comment:
ditto

Reply:
It has already been covered, here is just a function stripped out of the original code.